.github/workflows/pr_shelter.yml

```diff
@@ -77,7 +78,8 @@ jobs:
         docker exec $inclavare_test bash -c 'apt-get update -y && apt-get install -y wget gnupg;
         echo "deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main" | tee /etc/apt/sources.list.d/intel-sgx.list && wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -;
         apt-get update -y && apt-get install -y libsgx-dcap-quote-verify libsgx-dcap-default-qpl libsgx-dcap-ql-dev libsgx-uae-service;
-        cp -rf /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt'
+        cp -rf /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt;
+        cp -rf /root/inclavare-containers/${{ matrix.tag }}/enclave-tls/samples /usr/share/enclave-tls/samples'
```

Comment:
You need to `mkdir -p /usr/share/enclave-tls` in the docker container at first, Thanks.

Reply:
done, thx

Reply:
> done, thx

Approved